tests/hardening/5.1.3_cron_hourly_perm_ownership.sh

```diff
@@ -10,6 +10,8 @@ test_audit() {
     local test_user="testcrontabuser"
     local test_file="/etc/cron.hourly"
 
+    touch "$test_file"
```

Comment:
Are you sure quoted are required here ?

Reply:
no it's not required, but I have the habit to put quotes everywhere